tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/replication/RepeatStartBackupTest.java

```diff
@@ -156,8 +171,110 @@ public void testLoopStart() throws Exception {
 
          Assertions.assertEquals(0, errors.get());
       }
+   }
+
+   @Test
+   public void testAckManagerRepetition() throws Exception {
+
+      String queueName = "queue_" + RandomUtil.randomString();
+
+      server.getConfiguration().setMirrorAckManagerQueueAttempts(300000);
+      server.getConfiguration().setMirrorAckManagerRetryDelay(1000);
+      backupServer.getConfiguration().setMirrorAckManagerPageAttempts(300000);
+      backupServer.getConfiguration().setMirrorAckManagerRetryDelay(1000);

```

Comment:
Do the delays mean the test basically _has to_ take > 1second? Is that really needed? Similarly, given the delays, are the massive attempts necessary?